src/components/text-editor/text-editor.tsx

```diff
@@ -111,6 +112,16 @@ export class TextEditor implements FormComponent<string> {
     @Prop({ reflect: true })
     public allowResize: boolean = true;
 
+    /**
+     * Defines how the editor should look
+     * - `standard`: The default look of the editor
+     * - `minimal`: A more compact look of the editor, useful for
+     * displaying on mobile devices for instance space is limited.
+     * It will remove the toolbar, unless the editor is in focus.
+     */
+    @Prop({ reflect: true })
+    public ui?: 'standard' | 'minimal' = 'standard';
```

Comment:
I renamed these to keep them in sync with feed-items' `ui` types

Reply:
that's a good idea!